examples/CMakeLists.txt

```diff
@@ -53,4 +57,6 @@ elseif(MSVC)
     add_example(z_queryable windows/z_queryable.c)
     add_example(z_info windows/z_info.c)
     add_example(z_scout windows/z_scout.c)
+    add_example(z_ping windows/z_ping.c)
```

Comment:
These are not windows friendly :-)
Exclude them here if not ported for Windows.

Reply:
Since they are ported using pico's abstraction of mutex and condvar, they should work on any platform pico supports, so they are windows friendly (I had just forgotten to remove the include x) )